test/models/Stafftest.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Defines tests for the Student sailsjs model.
+ */
+
+"use strict";
+const helperIntegrationTests = require("./_helpers");
+
+describe("Staff model", function() {
+    let testData = {
+        associations: {},
+        record: {}
+    };
+
+    let testStaff = {
+        username: "TESTSTUDENT@DEWV.EDU",
+        firstName: "TEST",
+        lastName: "STAFF",        
+        isSlpInstructor: "TESTSLP"
```

Comment:
You declared this as a boolean in the staff models but here you gave it a string.  Might want to fix this before pulling it in. @Emreyildirim1284

Reply:
Upon our discussion, it is a apparent that the pure booleans I am considering don't fit as appropriately as a particular string.  that being TRUE or FALSE.  double check sails api to see if it will accept these as boolean accepters for mysql.  As I believe that mysql uses those values as pure booleans.